Core/src/org/sleuthkit/autopsy/geolocation/MapPanel.form

```diff
@@ -116,7 +120,9 @@
                 <Property name="borderPainted" type="boolean" value="false"/>
                 <Property name="focusPainted" type="boolean" value="false"/>
                 <Property name="requestFocusEnabled" type="boolean" value="false"/>
-                <Property name="rolloverEnabled" type="boolean" value="false"/>
+                <Property name="rolloverEnabled" type="boolean" editor="org.netbeans.modules.form.RADConnectionPropertyEditor">
```

Comment:
I'm not. This is my way of keeping them "false", since my IDE is removing these lines unless I manually specify "false" as user code. I'll just manually restore this file.